congress/metadata/G000578.yaml

```diff
@@ -0,0 +1,2 @@
+name: Florida House of Representatives
+link: http://www.myfloridahouse.gov/Sections/Representatives/representatives.aspx?SortField=district&SortDirection=asc&SessionId=73
```

Comment:
Ugh, sorry for this, I know this is a .gov, but state governments don't have the same guarantee of public domain as the federal government. I can't find a quick replacement for this one except for the (bad) avatar at https://twitter.com/Rep_Matt_Gaetz.